instrumentation-api-semconv/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/db/SqlClientAttributesExtractorBuilder.java

```diff
@@ -30,6 +31,7 @@
    * @param dbTableAttribute The {@link AttributeKey} under which the table extracted by the {@link
    *     SqlClientAttributesExtractor} will be stored.
    */
+  @CanIgnoreReturnValue
```

Comment:
The SDK repo ignores this suggestion -- should we do that too?

Reply:
I would be ok with that, but I also see some value to end users in adding these, @jack-berg any other thoughts?

Reply:
I don't have any particularly deep thoughts on the issue. The java sdk is just littered with builders and not ignoring this would require quite a lot of annotating. At the time I suppose I wasn't convinced that the annotation's value was worth all that effort.

Reply:
thx @jack-berg

@mateuszrzeszutek my vote would be to keep it. my guess is that next errorprone will add (or enable by default) a detector for ignored return values, which can be a useful check, and this will be needed.

Reply:
> my guess is that next errorprone will add (or enable by default) a detector for ignored return values, which can be a useful check, and this will be needed.

👍 
Let's keep it then, if it does provide some value.